### PR TITLE
[NAV2/ALL ENV] Split vars rabbitmq for use new rabbitmq servers

### PR DESCRIPTION
**Pull request by jmuller1984**

### CODE REVIEW

File: fabfile/component/kraken.py [hunk 1]
```diff
@@ -568,19 +568,21 @@ def delete_kraken_queue_to_rabbitmq(instance, apply_on='reverse'):
     else:
         abort("Bad 'apply_on' parameter value: {}".format(apply_on))
 
-    if env.rabbitmq_host_api == 'localhost':
-        host_string = env.roledefs['tyr_master'][0]
-    else:
-        host_string = env.rabbitmq_host_api
+    if env.rabbitmq_kraken_vhost == '/':
+	rabbitmq_vhost = '%2F'
```
Comment (jeremielate) [inline fabfile/component/kraken.py]: c'est mal indenté :slightly_smiling_face:

File: fabfile/component/kraken.py [hunk 2]
```diff
@@ -568,19 +568,21 @@ def delete_kraken_queue_to_rabbitmq(instance, apply_on='reverse'):
...
+    else
```
Comment (jeremielate) [inline fabfile/component/kraken.py]: `:`

File: fabfile/component/kraken.py [hunk 3]
```diff
@@ -568,19 +568,22 @@ def delete_kraken_queue_to_rabbitmq(instance, apply_on='reverse'):
     else:
         abort("Bad 'apply_on' parameter value: {}".format(apply_on))
 
-    if env.rabbitmq_host_api == 'localhost':
-        host_string = env.roledefs['tyr_master'][0]
+    if env.rabbitmq_kraken_vhost == '/':
+        rabbitmq_vhost = '%2F'
```
Comment (nonifier) [inline fabfile/component/kraken.py]: Are you sure you need to percent encode the `/` ? 
It seems useless at you define later one `env.rabbitmq_kraken_vhost = '/'`

Comment (jmuller1984) [reply fabfile/component/kraken.py]: It's for curl command. She doesn't accept //

File: fabfile/env/platforms.py [hunk 4]
```diff
@@ -184,25 +184,41 @@ def set_postgresql_database_host(db):
...
 env.rabbitmq_stats_port = 5672
-env.rabbitmq_stats_user = 'user'
-env.rabbitmq_stats_pass = 'pwd'
+env.rabbitmq_stats_user = 'guest'
+env.rabbitmq_stats_pass = 'guest'
 env.rabbitmq_stats_vhost = '/'
-
-
-env.stat_broker_exchange = 'stat_persistor_exchange'
+env.rabbitmq_stats_exchange = 'stat_persistor_exchange'
```
Comment (nonifier) [inline fabfile/env/platforms.py]: Are you sure the default value isn't `stat_persistor_exchange_topic` ??

Comment (nonifier) [reply fabfile/env/platforms.py]: https://github.com/CanalTP/navitia_deployment_conf/pull/707/files#diff-abc52033eca8627baa26c90da3776795L161

File: fabfile/component/kraken.py (hunk 3 above)
Comment (nonifier) [reply fabfile/component/kraken.py]: I've just run `curl -u <user>:<pass> par-vm182.srv.canaltp.fr:15672/api/queues/// ` and it does work.

**Pull request merged**